sdk/core/core-amqp/src/util/checkNetworkConnection.ts

```diff
@@ -10,25 +10,25 @@ import { logger } from "../log";
  * @internal
  */
 export function checkNetworkConnection(host: string): Promise<boolean> {
-  return new Promise((res) => {
-    logger.verbose("Calling dns.resolve to determine network connection status.");
-    resolve(host, function(err: any): void {
+  return new Promise((resolve) => {
+    logger.verbose("Calling dns.dnsResolve to determine network connection status.");
+    dnsResolve(host, function(err: any): void {
       if (err) {
         logger.verbose(
-          "Error thrown from dns.resolve in network connection check: '%s', %O",
+          "Error thrown from dns.dnsResolve in network connection check: '%s', %O",
           err.code || err.name,
           err
         );
         // List of possible DNS error codes: https://nodejs.org/dist/latest-v12.x/docs/api/dns.html#dns_error_codes
-        // Only when dns.resolve returns an error we expect to see when the network is down, resolve as 'false'.
+        // Only when dns.dnsResolve returns an error we expect to see when the network is down, dnsResolve as 'false'.
         if (err.code === CONNREFUSED || err.code === TIMEOUT) {
-          return res(false);
+          return resolve(false);
         }
       } else {
-        logger.verbose("Successfully resolved host via dns.resolve in network connection check.");
+        logger.verbose("Successfully dnsResolved host via dns.dnsResolve in network connection check.");
```

Comment:
I think the find/replace was a bit aggressive here. We probably want to keep the comment and logger text as it was throughout this promise and only change the symbols.

Reply:
Thank you for helping me spot this!